### DISCUSSION TITLE
Support for custom blocks

### DISCUSSION THREAD

**Discussion by elevatebart** [vuejs/language-tools#242] [Ideas]
In SFCs and in vetur one can support their own languages in custom blocks.
I would love a way to make docs support markdown, and i18n support json.

What do you think? Would this be roadmapable?

**Comment by johnsoncodehk** [comment]
No plan to support default language for third-party plugin's tags, this project intends to reduce the dependency with other plugins.

But we already supported `<xxx lang="json">`, `<xxx lang="md">`, is it enough for you?

**Reply by elevatebart** [reply]
It is exactly what I need.

💯 

Thank you
